C/common/filter_pipeline.cpp

```diff
@@ -405,3 +410,192 @@ void FilterPipeline::completeBranch()
 		m_branchActivations.notify_all();
 	}
 }
+
+/**
+ * Attach the debugger to the pipeline elements
```

Comment:
All of the method headers in this file have a description but many have no _@param_ or _@return_ descriptions.

Reply:
Fixed